plugins/convertPathData.js

```diff
@@ -408,34 +400,34 @@ function filters(path, params) {
             // Rounding relative coordinates, taking in account accummulating error
             // to get closer to absolute coordinates. Sum of rounded value remains same:
             // l .25 3 .25 2 .25 3 .25 2 -> l .3 3 .2 2 .3 3 .2 2
-            if (precision !== false) {
```

Comment:
removed that check, I don't think that's useful, precision=false would keep raw values?